solution/DecReprSeniorCoding.rb

```diff
@@ -0,0 +1,10 @@
+class DRSC
```

Comment:
Always use the class name related to file name. In this case, it should be `DecReprSeniorCoding`.
Ruby file name should be in snake case, in this case, `dec_repr_senior_coding.rb`.

Please apply the concept to its test suit as well krub.